interface-definitions/include/interface/dhcp-options.xml.i

```diff
@@ -14,6 +14,12 @@
         <help>Override system host-name sent to DHCP server</help>
       </properties>
     </leafNode>
+    <leafNode name="mtu">
+      <properties>
+        <help>Override interface MTU and get it from DHCP server</help>
```

Comment:
Can we respell this: `Use MTU value from DHCP server - ignore interface setting`

Reply:
> Can we respell this: `Use MTU value from DHCP server - ignore interface setting`

Sure. Done!